From dab2582f05979cd99a5937cb95a387cfed70bb8d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 16 Dec 2015 14:18:11 +0000
Subject: [PATCH 3/3] gtk3+wayland: start floating windows hidden

once they are visible we can't move them under wayland, so
start then hidden, move them and then show

these floating windows are very delicate.

Change-Id: Ia233c23f25cec453df79ef70cab6210dbdd280a0
---
 sfx2/uiconfig/ui/floatingrecord.ui  | 1 -
 svx/uiconfig/ui/colorwindow.ui      | 1 -
 svx/uiconfig/ui/floatingundoredo.ui | 1 -
 3 files changed, 3 deletions(-)

diff --git a/sfx2/uiconfig/ui/floatingrecord.ui b/sfx2/uiconfig/ui/floatingrecord.ui
index 653867a..9136b38 100644
--- a/sfx2/uiconfig/ui/floatingrecord.ui
+++ b/sfx2/uiconfig/ui/floatingrecord.ui
@@ -3,7 +3,6 @@
 <interface>
   <requires lib="gtk+" version="3.0"/>
   <object class="GtkWindow" id="FloatingRecord">
-    <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="no_show_all">True</property>
     <property name="border_width">6</property>
diff --git a/svx/uiconfig/ui/colorwindow.ui b/svx/uiconfig/ui/colorwindow.ui
index 71cda63..9482562 100644
--- a/svx/uiconfig/ui/colorwindow.ui
+++ b/svx/uiconfig/ui/colorwindow.ui
@@ -17,7 +17,6 @@
     <property name="icon_size">1</property>
   </object>
   <object class="GtkWindow" id="palette_popup_window">
-    <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="hexpand">True</property>
     <property name="vexpand">True</property>
diff --git a/svx/uiconfig/ui/floatingundoredo.ui b/svx/uiconfig/ui/floatingundoredo.ui
index 10491f8..66f30a6 100644
--- a/svx/uiconfig/ui/floatingundoredo.ui
+++ b/svx/uiconfig/ui/floatingundoredo.ui
@@ -3,7 +3,6 @@
 <interface>
   <requires lib="gtk+" version="3.0"/>
   <object class="GtkWindow" id="FloatingUndoRedo:border">
-    <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="hexpand">True</property>
     <property name="vexpand">True</property>
-- 
2.5.0

